core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -768,18 +768,16 @@ describe('Contributions and review component', () => {
       tick();
 
       expect(component.isReviewTranslationsTab()).toBeTrue();
-      expect(component.isReviewQuestionsTab()).toBeFalse();
```

Comment:
I think you can probably keep this assertion, it does no harm and is a reasonable check to make sure the calls are working correctly.